tsl/test/expected/dist_policy.out

```diff
@@ -0,0 +1,100 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+-- Need to be super user to create extension and add data nodes
+\c :TEST_DBNAME :ROLE_CLUSTER_SUPERUSER;
+CREATE OR REPLACE FUNCTION test_drop_chunks(job_id INTEGER)
+RETURNS VOID
+AS :TSL_MODULE_PATHNAME, 'ts_test_auto_drop_chunks'
+LANGUAGE C VOLATILE STRICT;
+-- Cleanup from other potential tests. This avoid additional notice
+-- messages when adding data nodes.
+SET client_min_messages TO ERROR;
+DROP DATABASE IF EXISTS data_node_1;
```

Comment:
why are you dropping databases here? this guarantees this is not gonna play well with other tests, you should drop databases you create at end of you script and choose a database name that does not conflict with existing scripts

Reply:
Yes,  I ran into CI issues with similar named data nodes that would fail intermittently and took a while to figure out what was going on. Suggest unique data node name for the test.